source/_docs/installation.markdown

```diff
@@ -11,55 +11,41 @@ redirect_from: /getting-started/installation/
 ---
 
 <p class='note'>
-Beginners should check our [Getting started guide](/getting-started/) first. This is for users that require advanced installations.
+Beginners should check our [Getting started guide](/getting-started/) first.
 </p>
 
-Home Assistant provides multiple ways to be installed. A requirement is that you have [Python 3.5.3 or later](https://www.python.org/downloads/) installed.
+Home Assistant provides multiple ways to be installed. The first start may take up to 20 minutes because the required packages will be downloaded and installed. The web interface will be served on `http://ip.add.re.ss:8123/` - replace `ip.add.re.ss` with the IP of the computer you installed it on.
 
 <p class='note warning'>
   Please remember to [secure your installation](/docs/configuration/securing/) once you've finished with the installation process.
 </p>
 
-## {% linkable_title Recommended options %}
+## {% linkable_title Recommended %}
 
-<div class="text-center hass-option-cards" markdown="0">
-  <a class='option-card' href='/getting-started/'>
-    <div class='img-container'>
-      <img src='/images/supported_brands/home-assistant.png' />
-    </div>
-    <div class='title'>Hass.io<br>(Beginner friendly)</div>
-  </a>
-  <a class='option-card' href='/docs/hassbian/installation/'>
-    <div class='img-container'>
-      <img src='/images/supported_brands/home-assistant.png' />
-    </div>
-    <div class='title'>Hassbian (for the Raspberry Pi)</div>
-  </a>
-  <a class='option-card' href='/docs/installation/docker/'>
-    <div class='img-container'>
-      <img src='/images/supported_brands/docker.png' />
-    </div>
-    <div class='title'>Docker</div>
-  </a>
-</div>
+These install options are fully supported by Home Assistant's documentation. For example, if a component requires that you install something to make it work on one of these methods then the component page will document the steps required.
+
+**Method**|**You have**|**Recommended for**
+:-----|:-----|:-----
+[Hass.io](https://www.home-assistant.io/hassio/installation/)|Raspberry Pi<br>VM|Anybody
+[Docker](https://www.home-assistant.io/docs/installation/docker/)|Docker|Anybody already running Docker
+[Hassbian](https://www.home-assistant.io/docs/hassbian/installation/)|Raspberry Pi|Those who want a more traditional Linux experience<BR>and either have experience with Linux, or intend to learn
```

Comment:
Drop the `<br>`, as users will visit on mobile etc, with different space available for lines.